src/include/sof/audio/cadence/aac_dec/xa_aac_dec_api.h

```diff
@@ -0,0 +1,417 @@
+/*
+ * Copyright (c) 2006-2020 Cadence Design Systems, Inc.
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+
+#ifndef __XA_AAC_DEC_API_H__
+#define __XA_AAC_DEC_API_H__
+
+/* aac_dec-specific configuration parameters */
+enum xa_config_param_aac_dec {
+  XA_AACDEC_CONFIG_PARAM_BDOWNSAMPLE          = 0,    /* Applicable only for aac*plus* libraries */
+  XA_AACDEC_CONFIG_PARAM_BBITSTREAMDOWNMIX    = 1,    /* Applicable only for aac*plus* libraries */
+  XA_AACDEC_CONFIG_PARAM_EXTERNAL_SAMPLERATE  = 2,
+#define XA_AACDEC_CONFIG_PARAM_EXTERNALSAMPLINGRATE XA_AACDEC_CONFIG_PARAM_EXTERNAL_SAMPLERATE // Renamed, maintained for backward comapatible
+  XA_AACDEC_CONFIG_PARAM_EXTERNALBSFORMAT     = 3,
+  XA_AACDEC_CONFIG_PARAM_TO_STEREO            = 4,
+  XA_AACDEC_CONFIG_PARAM_OUT_SAMPLERATE       = 5,
+#define  XA_AACDEC_CONFIG_PARAM_SAMP_FREQ           XA_AACDEC_CONFIG_PARAM_OUT_SAMPLERATE // Renamed, maintained for backward comapatible
```

Comment:
compatibility

Reply:
@plbossart I prefer to not touch this files.

Reply:
agree, I was just having fun